kubectl_plugins/bind-service/bind-service.go

```diff
@@ -0,0 +1,104 @@
+package main
```

Comment:
I assume `kubectl_plugins` here is just a place to start hacking on this and not a statement that this code should live where it currently is in this PR.

Precedent for this type of code in kubernetes: the scheduler lives in [`plugin/cmd/kube-scheduler`](https://github.com/kubernetes/kubernetes/tree/master/plugin/cmd/kube-scheduler).  How about `plugin/cmd/kubectl/<binary-name>` for these?